appeals/web/src/server/lib/mappers/appeal.mapper.js

```diff
@@ -168,8 +168,16 @@ export async function initialiseAndMapAppealData(
 				value: {
 					html: appealDetails.appellant
 						? `${appealDetails.appellant?.firstName + ' ' || ''}${
-								appealDetails.appellant?.lastName + '<br>' || ''
-						  }${appealDetails.appellant?.email || ''}`
+								appealDetails.appellant?.lastName || ''
+						  }${
+								appealDetails.appellant?.organisationName
+									? '<br>' + appealDetails.appellant?.organisationName
```

Comment:
It might be worth considering if there's more semantically meaningful HTML that could be used here, maybe an unordered list or definition list?